### PR TITLE
Fix two errors

### PR DESCRIPTION
**Pull request by xiaojun207**
1、Fix an error of "search result empty by undefined indexData.name"；
2、Fix the exception caused by empty returned data;

### CODE REVIEW

File: web/src/components/search/SearchResult.vue [hunk 1]
```diff
@@ -443,6 +443,11 @@ export default defineComponent({
       }
       searchLoading.value = true;
       const query = buildSearch(queryData);
+
+      if (!indexData.name) {
+        indexData.name = ""
```
Comment (hengfeiyang) [inline web/src/components/search/SearchResult.vue:448]: You should have `;` at the end to keep format.

File: web/src/components/search/SearchResult.vue [hunk 2]
```diff
@@ -478,6 +483,10 @@ export default defineComponent({
 
             // rerender the chart
             nextTick(() => {
+              if(!res.data.aggregations){
+                console.log("res.data.aggregations is null")
```
Comment (hengfeiyang) [inline web/src/components/search/SearchResult.vue:487]: You should have `;` at the end to keep format.
And you should have a blank space before `{` to keep format.

**Comment by xiaojun207** [comment]
Fix an other error, when selected index is none , the "select index component" will return old selected index

**Comment by hengfeiyang** [comment]
Please format your code, it need blank space after `)` or before `{`, and should add `;` at the end of line.

**Review by hengfeiyang — APPROVED** [review]

**Pull request merged**